fjord/feedback/utils.py

```diff
@@ -28,3 +35,23 @@ def clean_url(url):
     new_url = (parsed.scheme, parsed.hostname, parsed.path, None, None, None)
 
     return urlparse.urlunparse(new_url)
+
+
+POSSIBLE_EMOJIS_RE = re.compile(u'[\U00010000-\U0010ffff]')
+
+
+def convert_emoji(text):
+    """Takes unicode text and converts emoji characters
+
+    Emoji break MySQL, so we convert them into ascii.
+
+    """
+    # convert to unicodedata.name()
+    def _convert(match_char):
+        c = match_char.group(0)
+        try:
+            return unicodedata.name(c)
+        except ValueError:
+            # Throws a ValueError if the name doesn't exist.
+            return ''
```

Comment:
I thought about having this return "CRYING BECAUSE MYSQL SUCKS" because that's how I feel, but I didn't want to mess up a lot of feedback data and everything sucks--it's not just mysql.

I figured it was prudent to immortalize that in at least a pr comment.